### PR TITLE
Fix the hint colors when an account is hidden

### PR DESCRIPTION
**Pull request by kirizdev**
I don't really understand why the properties are named like this: "chat_gifSaveHint*". It doesn't really make sense. It was such a pain to find, lol :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ If you want to download a specific version of this theme <ins>DO NOT</ins> searc
 # Contributors
 I am grateful to all the people who have made a contribution of any size to this project. **Thank you**!
 
-- [@kirzidev](https://github.com/kirizdev)
+- [@kirizdev](https://github.com/kirizdev)
```
Comment (maximilionus) [inline README.md:81]: Oops, my bad, sorry for this.

**Comment by maximilionus** [comment]
Thanks for your contributions! Could you please elaborate on this problem - I'm kinda not getting where's this "hidden user" element? Screenshots would be greatly appreciated.

**Comment by kirizdev** [comment]
image
The text is hard to read so I decided to change the color. (Ignore the green text, I was testing)

**Review by maximilionus — APPROVED** [review]
Yep, that does fix the problem, thanks for this fix.

**Comment by kirizdev** [comment]
Glad to help!

**Pull request merged**